test/rpc/test_rpc_schema.py

```diff
@@ -0,0 +1,335 @@
+"""
+Test RPC schema validation
+"""
+from test.rpc.rpc_utils import rpc_call
+from test.util import devnet_in_background
+from unittest.mock import MagicMock, patch
+
+import pytest
+from starkware.starknet.public.abi import get_selector_from_name
+
+from starknet_devnet.blueprints.rpc.schema import _assert_valid_rpc_request
+from starknet_devnet.blueprints.rpc.structures.types import PredefinedRpcErrorCode
+from starknet_devnet.blueprints.rpc.utils import rpc_felt
+
+
+def _assert_call_not_raises_predefined_error(error_code: int):
+    assert error_code != PredefinedRpcErrorCode.INVALID_PARAMS.value
```

Comment:
Why this negative check?

Reply:
Oops, it was meant to check agains all possible values. Will fix that.